src/runtime/manifest-parser.pegjs

```diff
@@ -1728,6 +1747,87 @@ NumberedUnits
     });
   }
 
+// TODO(b/157605585): Annotations on policies.
```

Comment:
should this be a link to a parallel gh issue instead?

Reply:
We've been using b/ links in Kotlin code pretty comprehensively. I think the general decision was that you don't have to make parallel GitHub issues unless there's a good reason to (or you want to).

Reply:
oh, sorry, i missed that. it's good to know.